css-structure/css-structure_selectors-and-cascading/css/styles.css

```diff
@@ -49,92 +49,92 @@ nav li {
 /* -------------------------------------------------------- */
 
 /* 1. These styles should apply to the headline */
-FIX_ME {
+h1 {
   outline: 2px dashed red;
   font-size: 2rem;
   margin: 0 2rem 0 0;
   color: #2673bb;
 }
 
 /* 2. These styles should apply to all links */
-FIX_ME {
+a {
   outline: 2px dashed red;
   color: #ff00e6;
 }
 
 /* 3. These styles should apply to all external links (you can use the attribute selector) */
-FIX_ME {
+a[href^="http://"]{
   outline: 2px dashed red;
   color: #ff8000;
 }
 
 /* 4. These styles should apply to all links in the main navigation */
-FIX_ME {
+.nav a {
   outline: 2px dashed red;
   color: #ffffff;
   text-decoration: none;
 }
 
 /* 5. These styles should apply to the main navigation link corresponding to the current page */
-FIX_ME {
+.nav-a.current {
   outline: 2px dashed red;
   color: #22e781;
 }
 
 /* 6. These styles should apply to all links in the footer */
-FIX_ME {
+footer a {
   outline: 2px dashed red;
   color: inherit;
 }
 
 /* 7. These styles should apply to all paragraphs and all lists (please use a class selector) */
-FIX_ME {
+.text, .list{
   outline: 2px dashed red;
   color: #2c3f3f;
 }
 
 /* 8. These styles should apply to all "notes" */
-FIX_ME {
+.note{
   outline: 2px dashed red;
   font-size: 0.8em;
   color: #808080;
 }
 
 /* 9. These styles should apply to the welcome text */
-FIX_ME {
+welcome-text {
```

Comment:
welcome-text is this an id selector or class selector? CSS will not be able to identify the text itself, we should target by exact selector, for that we need to check, from the index.html we see that selector is id `#welcome-text {}`

```
 <p class="text" id="welcome-text">
          Welcome to your basic practice website! Enjoy your stay!
        </p>
```